lesson-3/script.js

```diff
@@ -0,0 +1,89 @@
+function userPortrait () {
+
+    let dateUser;
+    
+    function trueDate () {
+        return !dateUser || dateUser < 1900 || dateUser > 2005;
+    }
+
+    do {
+        dateUser = prompt('Enter please your birth date');
+
+        if (trueDate(dateUser)) {
+            alert('It is a pity that you did not want to enter your birth date...');
+        }
+    } while (trueDate(dateUser));
+
+
+    let currentDate = new Date().getFullYear() - dateUser;
+
+    let cityUser;
+
+    function trueCity () {
+        return !cityUser || cityUser == ' ';
```

Comment:
1. never use non-strict equality/inequality `==`,`!=` (strict equality/inequality  only `===`, `!==`) 
2. use trim() method to get rid of extra (or random) spaces around the line
```suggestion
        return !cityUser.trim();
```